apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/index.php

```diff
@@ -15,14 +15,23 @@ function render_template_block( $attributes ) {
 	if ( ! isset( $attributes['templateId'] ) || ! is_int( $attributes['templateId'] ) ) {
 		return;
 	}
+
+	$parent_post_id = get_the_ID();
+	$template = get_post( $attributes['templateId'] );
+
 	$align = isset( $attributes['align'] ) ? ' align' . $attributes['align'] : '';
-	$post  = get_post( $attributes['templateId'] );
-	setup_postdata( $post );
 
-	$content = '<div class="template' . $align . '">'
-		. apply_filters( 'the_content', get_the_content() )
-		. '</div>';
+	setup_postdata( $template );
+	ob_start();
+	?>
 
+		<div class="template-part<?php echo $align; ?>">
+			<?php echo apply_filters( 'the_content', get_the_content() );  ?>
+		</div><!-- .template-part -->
+
+	<?php
+	$content = ob_get_clean();
```

Comment:
what's the point of `ob_start()` here? is it to handle string interpolation as HTML vs. as a PHP string?

```php
setup_postdata( $template );
$content = sprintf(
	'<div class="template-part%s">%s</div><!-- .template-part -->',
	$align,
	apply_filters( 'the_content', get_the_content() )
);
wp_reset_postdata();
```